.github/workflows/test_hwlib_debian_build.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build hwlib Rust package with sbuild
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - client/**
+      - Cargo.*
+      - ".github/workflows/test_hwlib_debian_build.yaml"
+  pull_request:
+    branches: [ main ]
+    paths:
+      - client/**
+      - Cargo.*
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux, large, noble, x64]
+    defaults:
+      run:
+        working-directory: client/hwlib
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v4
+    - name: Download dependencies
+      run: |
+        sudo apt update
```

Comment:
Add `set -xeu` to make sure that you:
- see what is executed with trace (x)
- fail if anything fails in a simple statement (e)
- fail if there's an undefined variable being referenced (u)

Reply:
I've added the `set -xeu` to steps with multiple commands